lmdeploy/vl/model/deepseek.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from typing import List
+
+import torch
+from PIL.Image import Image
+
+from lmdeploy.vl.model.base import VisonModel
+
+
+def check_deepseek_vl_install():
+    """check deepseek_vl install."""
+    try:
+        import deepseek_vl  # noqa: F401
+    except ImportError:
+        raise ImportError(
+            'To use DeepSeekVLModel, please install deepseek_vl by '
+            'pip install git+https://github.com/deepseek-ai/DeepSeek-VL.git')
+
+
+class DeepSeekVisionModel(VisonModel):
+    """Qwen vision model."""
+
+    def __init__(self, model_path, device='cuda:0'):
+        self.model_path = model_path
+        self.device = device
+        self.build_model()
+
+    def build_model(self):
+        check_deepseek_vl_install()
+        from deepseek_vl.models import VLChatProcessor
+        from transformers import AutoModelForCausalLM
+        with torch.device('cpu'):
+            model = AutoModelForCausalLM.from_pretrained(
+                self.model_path, trust_remote_code=True)
```

Comment:
may use `init_empty_weights` to accelerate loading

Reply:
tried. But seemed the output of the model would be wrong.

Reply:
Accelerating loading model is very important. Please investigate